client/lib/plans/index.js

```diff
@@ -348,18 +348,24 @@ export function calculateMonthlyPriceForPlan( planSlug, termPrice ) {
 }
 
 export function calculateMonthlyPrice( term, termPrice ) {
-	let divisor;
+	const divisor = getBillingMonthsForTerm( term );
+	return parseFloat( ( termPrice / divisor ).toFixed( 2 ) );
+}
+
+export function getBillingMonthsForPlan( planSlug ) {
+	return getBillingMonthsForTerm( getPlan( planSlug ).term );
+}
```

Comment:
Doesn't look like anything is using this function - do you need to add it?

Reply:
Thank you! I forgot to mention this function in PR description but it's going to be used :-)